atom/browser/api/lib/dialog.coffee

```diff
@@ -108,6 +109,7 @@ module.exports =
 
     binding.showMessageBox messageBoxType,
                            options.buttons,
+                           options.defaultButtonIndex,
```

Comment:
I think the new option should be called `defaultId` to match the way we call the cancel button index (`cancelId`).